test/ex_doc/language/erlang_test.exs

```diff
@@ -789,12 +789,12 @@ defmodule ExDoc.Language.ErlangTest do
 
     test "OTP type", c do
       assert autolink_spec(~S"-spec foo() -> sets:set().", c) ==
-               ~s|foo() -> <a href="https://www.erlang.org/doc/man/sets.html#type-set">sets:set</a>().|
+               ~s|foo() -> <a href="https://www.erlang.org/doc/man/sets.html#t:set/0">sets:set</a>().|
     end
 
     test "OTP private type", c do
       assert autolink_spec(~S"-spec foo() -> array:array_indx().", c) ==
-               ~s|foo() -> <a href="https://www.erlang.org/doc/man/array.html#type-array_indx">array:array_indx</a>().|
+               ~s|foo() -> array:array_indx().|
```

Comment:
This one "lost" its link to https://www.erlang.org/doc/apps/stdlib/array.html#t:array_indx/0